charts/datadog/values.yaml

```
@@ -134,6 +134,8 @@ datadog:
   clusterChecks:
     # datadog.clusterChecks.enabled -- Enable the Cluster Checks feature on both the cluster-agents and the daemonset
     enabled: true
+    # datadog.clusterChecks.shareProcessNamespace -- Set the process namespace sharing on the cluster checks agent
+    shareProcessNamespace: true
```

since in `corev1.PodSpec` the default value is false, it makes sens to also set to false the default value here.

```suggestion
    shareProcessNamespace: false
```